### ISSUE TITLE
Forking sometimes fails without any explanation

### ISSUE THREAD

**Issue by pfrazee** [beakerbrowser/beaker#1734]
This seems to mostly happen on readonly drives

**Comment by JerryGreen** [comment]
Yeah, I've just tried to copy your personal hyperdrive. It doesn't appear in "My Drives". It seems it conflicts with "host this drive" feature. When I've started hosting your drive, I'm no more able to fork/copy your drive. If I stop hosting your drive then the fork/copy feature will just enable the hosting back.

There's some conflicts between "host this drive" and "fork/copy".
